pkg/cvo/cvo_scenarios_test.go

```diff
@@ -2835,6 +2861,22 @@ func TestCVO_VerifyUpdatingPayloadState(t *testing.T) {
 	}
 }
 
+// verifyCVSingleUpdate ensures that the only object to be updated is a ClusterVersion type and it is updated only once
+func verifyCVSingleUpdate(t *testing.T, actions []clientgotesting.Action) {
+	var count int
+	for _, a := range actions {
+		if a.GetResource().Resource != "clusterversions" {
+			t.Fatalf("an resource of type %s was accessed/updated", a.GetResource().Resource)
```

Comment:
nit: "an resource" -> "resource" .

I was wondering if we could also log the name, but [the generic `Action`][1] doesn't have a `GetName()`.  Although I also see the `GetVerb() != "get"` below.  Maybe we should be explicit about the verbs that count as updates?  That would be `create`, `update`, and `delete`?  `patch`?  I dunno.

[1]: https://pkg.go.dev/k8s.io/client-go/testing#Action
And maybe we can log the name as well, to make this a bit easier to debug if it happens?

Reply:
Anything which is not a `get` would be a mutating change. So I think this is good enough. I've added the entire object into the failure log to help with debugging.